spec/features/idv/analytics_spec.rb

```diff
@@ -10,25 +10,25 @@
   let(:happy_path_events) do
     {
       'IdV: intro visited' => {},
-      'IdV: doc auth welcome visited' => { flow_path: 'standard', step: 'welcome', step_count: 1 },
-      'IdV: doc auth welcome submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'welcome', step_count: 1 },
-      'IdV: doc auth agreement visited' => { flow_path: 'standard', step: 'agreement', step_count: 1 },
-      'IdV: doc auth agreement submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'agreement', step_count: 1 },
-      'IdV: doc auth upload visited' => { flow_path: 'standard', step: 'upload', step_count: 1 },
-      'IdV: doc auth upload submitted' => { success: true, errors: {}, destination: :document_capture, flow_path: 'standard', step: 'upload', step_count: 1 },
-      'IdV: doc auth document_capture visited' => { flow_path: 'standard', step: 'document_capture', step_count: 1 },
+      'IdV: doc auth welcome visited' => { flow_path: 'standard', step: 'welcome', step_count: 1, native_camera_a_b_testing_enabled: false, native_camera_only: false },
```

Comment:
Is this at risk of being flakey if future test runs generate a UUID which results in a different bucket assignment? Or are we stubbing the behavior somewhere?

Reply:
By default the test is not enabled, so this shouldn't have any flakiness, but I explicitly force it to be disabled in 5a371e6aa